src/main/java/com/spring/app/RestExceptionHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.spring.app;
+
+import org.springframework.dao.InvalidDataAccessResourceUsageException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+/**
+ * Exception handler
+ * Return a message to client to explain problem
```

Comment:
Let's write it in more describable way. It doesn't return a message, it's an exception handler